<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DHCP assigned addresses</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="serverField">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>111</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="package_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QListWidget" name="hostField">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>141</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="addressField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>121</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>MAC address</string>
    </property>
   </widget>
   <widget class="QListWidget" name="macField">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>131</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QListWidget" name="statusField">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>111</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="version_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dynamic</string>
    </property>
   </widget>
   <widget class="QListWidget" name="dynamicField">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>111</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>130</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disableButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="staticButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Make static</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
